src/skeleton/mod.rs

```diff
@@ -346,6 +355,18 @@ fn ignore_all_members_except(manifests: &mut [ParsedManifest], member: String) {
         .and_then(|toml| toml.contents.get_mut("workspace"))
         .and_then(|workspace| workspace.get_mut("members"))
     {
-        *members = toml::Value::Array(vec![toml::Value::String(member)]);
+        *members = toml::Value::Array(vec![toml::Value::String(member.clone())]);
     }
+
+    // Remove manifest(s) with package name equal to `member`
```

Comment:
I think it's the opposite, isn't it?
We are *keeping* the manifests that don't have a package name or whose package name is *equal* to `member`.

Reply:
Of course, the comment is opposite, I'll fix it.